src/librustc/ty/util.rs

```diff
@@ -319,7 +384,23 @@ impl<'tcx> TyCtxt<'tcx> {
                     } else {
                         break;
                     }
-                },
+                        },
```

Comment:
Stray change?

Reply:
so this bit is goofy, I think its an artifact, at least in emacs, of the match guard starting on its own line ...

Reply:
ah but I didn't even touch the match guard. Okay reverted this.